options.go

```diff
@@ -6,6 +6,12 @@ import (
 
 type Option func(r hasOptions)
 
+func ID(id string) Option {
+	return func(r hasOptions) {
+		r.addLabel("as_id", id)
```

Comment:
Not having Allure Enterprise doesn't allow me to test this, but if I look at other repositories, it seems the label should be in capital : https://javadoc.io/static/io.qameta.allure/allure-java-commons/2.13.9/io/qameta/allure/AllureId.html
Can you please check on your side and confirm?

Reply:
Hi Maxime,
Yes, I use Allure Enterprise and I need to link test cases in Allure and in my golang suites, that's why I use AS_ID label.
I've checked what you asked and found that it works both ways ('as_id' and 'AS_ID'), so I've changed it to 'AS_ID' like in documentation. 
Also, I've added a label 'Tags'. I use labels 'Tag' and 'Tags' in my golang tests because without them, all tags on Allure server will be erased.